tests/queries/0_stateless/03036_reading_s3_archives.reference

```diff
@@ -0,0 +1,38 @@
+1	Str1	03036_archive1.zip	test/03036_archive1.zip
```

Comment:
to be consistent with `file` function/engine,  `_file` should return name of the file from inside the zip and `_path` should be `archive_path::_file`